app/requests_handlers.py

```diff
@@ -572,1083 +366,222 @@ class RequestCodeConfirmAddressHHNI(RequestCodeCommon):
                 session['attributes']['region'] = uprn_return[0]['region']
                 session.changed()
                 raise HTTPFound(
-                    request.app.router['RequestCodeEnterMobileHHNI:get'].
-                    url_for())
+                    request.app.router['RequestCodeEnterMobile:get'].
+                    url_for(request_type=request_type, display_region=display_region))
             except ClientResponseError as ex:
                 if ex.status == 404:
```

Comment:
Ticket already exists - this will be converted to 'ypur address is in Scotland' and similar, once updated AIMS provides country codes https://collaborate2.ons.gov.uk/jira/browse/CR-730